AGPs/AGP-CRDAO.md

```diff
@@ -0,0 +1,60 @@
+---
+AGP: N/A
+Title:  Community Rewards DAO
+Author: Aaron Foster (`@pythonpete32`) & Burrrata (`@burrrata`)
+Status: Stage III
+Track: Finance
+Created: 2019-10-01
+---
+
+# AGP X: Community Rewards DAO
+
+## Address of the transfer recipient
+
+TBD, but will be the Vault of the CRDAO
```

Comment:
Will this be added before this AGP is frozen today? If not please specify one person or entity that will provide the address

Reply:
Thank you for catching this. Added a recipient address.